packages/mobile/src/walletConnect/request.ts

```diff
@@ -29,14 +36,56 @@ export function* handleRequest({ method, params }: { method: string; params: any
   yield call(unlockAccount, account)
 
   switch (method) {
-    case SupportedActions.eth_signTransaction:
-      return (yield call(wallet.signTransaction.bind(wallet), params[0])) as EncodedTransaction
+    case SupportedActions.eth_signTransaction: {
```

Comment:
Should we normalize eth_send and eth_sign the same way?

Reply:
Yes we should, I'll open a PR applying the same logic to both.